Mathlib/Combinatorics/SimpleGraph/Maps.lean

```diff
@@ -671,4 +671,56 @@ noncomputable def overFinIso (hc : Fintype.card V = n) : G ≃g G.overFin hc :=
 
 end Finite
 
+section CrossEdges
+
+variable (K : Set V)
+
+/-- The graph formed by retaining only those edges from `K` to `Kᶜ`. -/
+def crossEdges : SimpleGraph V where
+  Adj a b := G.Adj a b ∧ ¬(a ∈ K ↔ b ∈ K)
```

Comment:
Could [Xor'](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Init/Logic.html#Xor') be helpful here?